src/core/Object3D.js

```diff
@@ -662,6 +668,20 @@ Object.assign( Object3D.prototype, EventDispatcher.prototype, {
 
 		}
 
+		// SkinnedMesh specific
+
+		if ( this.skeleton !== undefined ) {
```

Comment:
I can wait until this gets merged and i can try to propose moving and cleanup these after maybe?